apps/designer/app/canvas/features/wrapper-component/wrapper-component.tsx

```diff
@@ -40,19 +40,20 @@ const ContentEditable = ({
 
 type WrapperComponentDevProps = {
   instance: Instance;
-  css: CSS;
+  breakpoints: Array<Breakpoint>;
   children: Array<JSX.Element | string>;
   onChangeChildren?: OnChangeChildren;
 };
 
 export const WrapperComponentDev = ({
   instance,
-  css,
+  breakpoints,
```

Comment:
Is this prop used? By looking at the diff, seems like not used

Reply:
yep, removing